apps/mobile/src/features/balances/stacks/stacks-balance.tsx

```diff
@@ -22,18 +26,24 @@ export function StacksTokenBalance({ availableBalance, fiatBalance }: StacksToke
       protocol="nativeStx"
       fiatBalance={fiatBalance}
       availableBalance={availableBalance}
+      {...rest}
     />
   );
 }
 
-export function StacksBalance() {
+interface StacksBalanceProps {
+  onPress?(): void;
```

Comment:
Same as `BitcoinBalance` do we need this `onPress` ? Apologies if we do and you mentioned it before

Reply:
Ah I remember it's related to this now right? 

https://github.com/leather-io/mono/pull/984/files#r2004902921

Reply:
The onPress is definitely used, all those asset items need to be interactive in many flows.